### ISSUE TITLE
Manual Proof of Concept of Techniques

### ISSUE THREAD

**Issue by farrowking37** [farrowking37/Capstone#8]
Once I have determine what techniques I want to explore further, I will create a manual proof of concept video demonstrating each of these techniques.

**Comment by farrowking37** [comment]
I have completed two manual proofs of concept, demonstrating simple techniques for data hiding. One using Least Significant Bit steganography and another using NTFS Alternative data streams. You can find the videos for these at the following links:

[Least Significant Bit Stego in JPG images](https://drive.google.com/file/d/1sCZs9f4zv1RhukvIpD4MmkJ5DRGo-jOb/view?usp=sharing)
[NTFS Alternative Data Streams](https://drive.google.com/file/d/1pkqEMRWcjoqBwm4-fXX4ClX_rkqkEkMN/view?usp=sharing)

**Comment by farrowking37** [comment]
I have completed another proof of concept, this time demonstrating a method to hide files within .zip archives from the default Windows 10 explorer view. I also discussed why this method may not hide the files from other archiving tools.

There are two parts. The first shows the method, while the second discusses the behavior on other platforms.

[.zip Data Hiding Part 1](https://drive.google.com/file/d/1SyiyVdYKbEnDFrc15svcrYiDD5XZlgPu/view)
[.zip Data Hiding Part 2](https://drive.google.com/file/d/1t9ZV7dksK3EnM10x1CpA5ph_I5I6Y7RW/view)
